src/ORM/DataList.php

```diff
@@ -1946,7 +1946,8 @@ public function setByIDList($idList)
      */
     public function getIDList()
     {
-        $ids = $this->column("ID");
+        // Disabling sort improves performance
+        $ids = $this->sort(null)->column("ID");
```

Comment:
As per the PHPDoc for this method, the sort order is already explicitly noted as not being respected so that makes this 100% safe to do.

Reply:
> As per the PHPDoc for this method, the sort order is already explicitly noted as not being respected so that makes this 100% safe to do.

Anyone relying on the order of values being returned from this method is not using the method the way it was intended, as clearly indicated by the PHPDoc.

Reply:
As discussed offline, deprecate this method

Reply:
Done - also deprecated it from other list classes.